src/main/java/org/sonar/plugins/csharp/CSharpSensor.java

```diff
@@ -67,6 +67,10 @@
 import org.sonar.api.utils.command.Command;
 import org.sonar.api.utils.command.CommandExecutor;
 import org.sonar.api.utils.command.StreamConsumer;
+import org.sonar.plugins.csharp.sarif.SarifParser;
+import org.sonar.plugins.csharp.sarif.SarifParser01And04;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.plugins.csharp.sarif.SarifParser01And04'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)